requirements.txt

```diff
@@ -1,2 +1,3 @@
+httpx<0.28
 requests>=2.18
 openai==1.27.0
```

Comment:
I believe the fix is to use a later `openai` version instead.

Reply:
Yes, sorry for opening a PR here rather than on my fork, my bad 😅